README.md

```diff
@@ -93,6 +93,30 @@ First, update the repository (assuming you have `master` checked out):
 Look carefully for errors with `git pull`, and reset locally modified files if
 necessary.
 
+## Upgrading Instructions 
+
+*__Only necessary while upgrading from upgrading from v1.3.2.0.0.0.0.0.0.1 or below.__*
```

Comment:
"from v1.3.2 or below" (pretend like the other one doesn't exist. It was a joke)

Reply:
Noted